src/app/RouteConfig.ts

```diff
@@ -0,0 +1,61 @@
+interface PreviousActivityConfig {
+  prevStage: number;
+  prevActivity: number;
+}
+
+interface Routes {
+  [componentName: string]: PreviousActivityConfig
+}
+
+export const ActivityRoutes: Routes = {
+  HighlightActivityComponent: {
+    prevStage: 1,
+    prevActivity: 0
+  },
+
+  PcpolicyComponent: {
+    prevStage: 1,
+    prevActivity: 1
+  },
+
+  PicturePuzzleComponent: {
+    prevStage: 1,
+    prevActivity: 2
+  },
+
+  MalariaLifeCycleComponent: {
+    prevStage: 1,
+    prevActivity: 3
+  },
+
+  DragdropComponent: {
+    prevStage: 2,
+    prevActivity: 2
+  },
+
+  OddOneOutComponent: {
+    prevStage: 2,
+    prevActivity: 3
+  },
+
+  MatchmedsComponent: {
+    prevStage: 3,
+    prevActivity: 1
+  },
+
+  MemoryGameComponent: {
+    prevStage: 3,
```

Comment:
Are the values for prevStage correct? It doesn't look like it is

Reply:
I made a small change. Please see again.